gamelogic/player.go

```diff
@@ -1,25 +1,39 @@
 package gamelogic
 
 import (
+	"encoding/json"
 	"fmt"
 )
 
 type Player struct {
 	Name         string
-	input        UserInput
+	Input        UserInput
```

Comment:
i'm not sure i agree with this change. Other object's shouldn't have to worry about how player input works, they just interface directly with the player.

Reply:
talked over slack. Will privatize and expose wrapper methods around on Player so other modules can call Input methods